.gitignore

```diff
@@ -5,3 +5,4 @@ Corefile
 build/
 release/
 vendor/
+db.*
```

Comment:
What is this? This doesn't seem like it is necessary.

Reply:
Removed